src/main/java/org/fundacionjala/sfdc/pageobjects/common/SFCommonObjects.java

```diff
@@ -1,21 +1,27 @@
 package org.fundacionjala.sfdc.pageobjects.common;
 
+import org.fundacionjala.core.common.Base;
+import org.fundacionjala.core.driver.DriverManager;
+import org.fundacionjala.core.util.PropertiesManager;
 import org.fundacionjala.sfdc.util.CommonActions;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.util.CommonActions'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)